learn/documents/index.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+section: Learn
+title: Manual & Documents
+---
+		<div class="row">
+      <div class="col-md-2">&nbsp;</div>
+			<div class="col-md-8">
+        <h2>Using Core Open Rails</h2>
+        <ul>
+          <li><a href="https://open-rails.readthedocs.io/en/latest/" target="_blank">User Manual (Testing Version)</a></li>
+          Each stable and testing version includes a copy of the Operating Manual.<br />
+          Also you can <a href='{% include file_url.html file="testing_manual" %}'>Download a PDF edition</a>
+          <li><a href="https://static.openrails.org/files/OpenRails-Testing-Installation%20Guide%20En.pdf" target="_blank">Installation Guide</a></li>
+          <li><a href="https://coalstonewcastle.com.au/physics/open-rails-support/" target="_blank">Keyboard and Camera Controls v4</a></li>
+          <li><a href="https://static.openrails.org/files/Keyboard_Layout_DE_V1.3.1e.pdf" target="_blank">Keyboard Layout (DE) V1.3.1e</a></li>
+        </ul>
+        <h2>Using Non-Core (Contributed) Programs</h2>
+        <ul>
+          <li><a href="https://static.openrails.org/files/Content%20checker.pdf" target="_blank">Content Checker Manual</a></li>
+          <li><a href="https://static.openrails.org/files/timetable-editor_de.pdf" target="_blank">Timetable Editor (DE)</a></li>
+          <li><a href="https://static.openrails.org/files/timetable-editor_en.pdf" target="_blank">Timetable Editor (EN)</a></li>
+          <li><a href="https://static.openrails.org/files/ORTS_Trackviewer_manual.pdf" target="_blank">Trackviewer Manual</a></li>
```

Comment:
These URLs will be old versions of the files and everything in the repository should refer to the `OpenRails-Testing-` version of static files to stay up to date

Reply:
Added prefix for items from Documentation folder too.